README.rst

```diff
@@ -263,9 +226,7 @@ However, Rekall profiles have to be created for each kernel version, and therefo
 update is made to the kernel, the profile has to be re-generated, thus it's a bit less stable
```

Comment:
> update is made to the kernel

The beginning of this sentence is confusing.
(from a newcomer point of view): what are you talking about, `the kernel` ?